src/main/java/io/github/protonmc/proton/module/building/StainedPlanksModule.java

```diff
@@ -21,6 +25,15 @@ public void commonInit() {
         ModuleBlocks.register();
     }
 
+    @Override
+    public void registerResources(ResourceHandler resourceHandler) {
+        // i'm sorry
+        for (Field f : ModuleBlocks.class.getFields()) {
```

Comment:
replace this with something similar to the varianthandler call below, this was only temporary...